src/app/(auth)/signin.tsx

```diff
@@ -27,74 +44,143 @@ import { resetNavigationStack } from "@/src/lib/resetNavigationStack";
+      />
+
+      <SlidingDot
+        data={tutorialData}
+        scrollX={scrollX}
+        dotSize={10}
+        containerStyle={{ position: "relative" }}
+        dotStyle={{
+          backgroundColor: colorScheme === "dark" ? colors.white : colors.black,
+        }}
+        marginHorizontal={10}
+        slidingIndicatorStyle={{
+          backgroundColor: colorScheme === "dark" ? colors.white : colors.black,
+        }}
+      />
+
+      <Text className="mb-3 w-2/3 text-center text-xs text-stone-400">
+        By continuing, I agree to the {"\n"}
+        <Link href="/">
+          <Text className="text-xs text-stone-400 underline">
+            Terms & Conditions
+          </Text>
+        </Link>{" "}
+        and{" "}
+        <Link href="/">
+          <Text className="text-xs text-stone-400 underline">
+            Privacy Policy
+          </Text>
+        </Link>
+      </Text>
+
+      {Platform.OS === "ios" && (
+        <>
+          <AuthButton
+            icon={<IconBrandAppleFilled fill={colorScheme === "dark" ? colors.white : colors.black} color={colorScheme === "dark" ? colors.white : colors.black} strokeWidth={0} size={24} />}
+            text="Continue with Apple"
+            onPress={() => { }}
+          />
+          <AuthButton
+            icon={<Image className="h-5 w-5 rounded-[20px]" source={require("../../../assets/images/google.png")} />}
+            text="Continue with Google"
+            onPress={() => { }}
+          />
+          <Pressable className="mt-2">
+            <Text className="text-base font-semibold">
+              Sign up or log in with Email
+            </Text>
+          </Pressable>
+        </>
+      )}
+      {Platform.OS === "android" && (
+        <>
+        <AuthButton
+          icon={<Image className="h-5 w-5 rounded-[20px]" source={require("../../../assets/images/google.png")} />}
+          text="Continue with Google"
+          onPress={() => { }}
         />
-      </View>
-      <View style={[styles.verticallySpaced, styles.mt20]}>
-        <Button
-          title="Sign in"
-          disabled={loading}
-          onPress={() => signInWithEmail()}
+        <AuthButton
+          icon={<Icon icon={IconMail} lightColor={colors.black} darkColor={colors.white} strokeWidth={2.2} />}
+          text="Continue with Email"
+          onPress={() => { }}
         />
-      </View>
+      </>
+      )}
     </View>
   );
 }
 
-const styles = StyleSheet.create({
-  container: {
-    marginTop: 40,
-    padding: 12,
-  },
-  verticallySpaced: {
-    paddingTop: 4,
-    paddingBottom: 4,
-    alignSelf: "stretch",
-  },
-  mt20: {
-    marginTop: 20,
-  },
-});
+interface AuthButtonProps {
+  icon: JSX.Element;
+  text: string;
+  onPress: () => void;
+}
+
+
+function AuthButton({ icon, text, onPress }: AuthButtonProps) {
+  return (
+    <Pressable
+      className="mt-2 flex w-11/12 flex-row items-center justify-center rounded-xl border-2 p-2 dark:border-stone-300"
```

Comment:
Do not use `w-11/12`. Just make them full width with margin (or putting padding on parent element would be even better).

Also buttons do not look like they are as tall as in the design.